app/routes.test.js

```diff
@@ -158,20 +166,23 @@ describe('routes', () => {
   });
 
   describe('GET /solutions/compare/document', () => {
-    it('should call getDocument with the correct params', async () => {
-      getDocument.mockResolvedValue({ data: createReadStream(path.resolve(__dirname, 'data.pdf')) });
-      return request(setUpFakeApp())
-        .get('/solutions/compare/document')
-        .expect(200)
-        .then(() => {
-          expect(getDocument.mock.calls.length).toEqual(1);
-          expect(getDocument).toHaveBeenCalledWith({
-            endpoint: `${config.documentApiHost}/api/v1/documents/compare-solutions.xlsx`,
-            logger,
-          });
-          getDocument.mockReset();
+    it('should call getDocumentByFileName with the correct params when the user is authorised', () => request(setUpFakeApp())
+      .get('/solutions/compare/document')
+      .then(() => {
+        expect(documentController.getDocumentByFileName.mock.calls.length).toEqual(1);
+        expect(documentController.getDocumentByFileName).toHaveBeenCalledWith({
+          res: expect.any(Object),
```

Comment:
what's this about? and does it add any value

Reply:
no but without it the test will fail

Reply:
also this was in the other PR too 😄

Reply:
ok fair enough 👍  - if the test needs it then lets leave it in